_includes/layouts/base.njk

```diff
@@ -0,0 +1,47 @@
+---
+nav_links:
+ - name: link1
+   url: /link1
+ - name: link2
+   url: /link2
+ - name: link3
+   url: /link3
+---
+
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>{{ title or metadata.title }}</title>
+    <meta name="description" content="{{ description or metadata.description }}">
+    <link rel="stylesheet" href="{{ 'style.css' | url }}">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <header>
+      <a href="/" class="page-header__logo" title="Go home">
+          <img src="/img/avatar.png" alt="" width="48">
+      </a>
+      <h1 class="home"><a href="{{ '/' | url }}">{{ metadata.title }}</a></h1>
+
+      <ul class="nav justify-content-center">
+		{% for link in nav_links %}
+		<li class="nav-item{% if link.url == page.url %} nav-item-active{% endif %}">
+		  <a class="nav-link" href="{{ link.url | url }}">{{ link.name }}</a>
+		</li>
+		{% endfor %}
+	  </ul>
+    </header>
+
+    <main>
+      {{ content | safe }}
```

Comment:
This layout has our HTML head info and nav bar and will be used on every page.  The specific page content is injected here.